.github/workflows/e2e.install.yaml

```diff
@@ -0,0 +1,34 @@
+name: E2E - Chart Install
+
+on:
+  pull_request:
+    paths:
+      - "**"
+      - "!**.md"
+  push:
+    branches:
+      - feat/*
+  workflow_dispatch:
+
+jobs:
+  Install:
+    name: Chart Install
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build docker images for E2E testing
+        uses: ./.github/build-for-e2e
+
+      - name: Run E2E Test
+        uses: apache/skywalking-infra-e2e@main
+        with:
+          e2e-file: $GITHUB_WORKSPACE/test/e2e/cases/chart-install/e2e.yaml
```

Comment:
Is that possible to just use `test/e2e/cases/chart-install/e2e.yaml`?

Reply:
Yup, it it. I'm going to create a composite GitHub action to achieve this goal.

Reply:
But for this PR, this configuration works well. I will refactor them in the later PR.